### ISSUE TITLE
[bug] [www] missing aria-label in many components

### ISSUE THREAD

**Issue by mehdibha** [mehdibha/rcopy#125]
### Describe the bug

in www, many logs for missing aria-label

### Affected component/components

none

### How to reproduce

Check all the pages

### System Info

```bash
macos
```

### Before submitting

- [x] I've made research efforts and searched the documentation
- [x] I've searched for existing issues
